libs/utils/analysis/latency_analysis.py

```diff
@@ -775,7 +775,7 @@ def _getTaskData(self, task):
         else:
             raise ValueError("Task must be either an int or str")
 
-        task_label = "{}: {}".format(task_pid, ', '.join(task_names))
+        task_label = "{}: {}".format(task_pid, task_names)
```

Comment:
Please can you also rename `task_names` to `task_name`.

Reply:
Sure. I sent another PR for this.

But I want to check if there have updated patch, can I reuse this PR? I tried to "git push" to the same branch, but it failed.

Reply:
You can normally just amend the commit and do a `git push --force`